pdns/dnsdistdist/dnsdist-lua-ffi.cc

```diff
@@ -512,6 +512,39 @@ void dnsdist_ffi_dnsquestion_set_tag_raw(dnsdist_ffi_dnsquestion_t* dq, const ch
   dq->dq->setTag(label, std::string(value, valueSize));
 }
 
+void dnsdist_ffi_dnsquestion_set_requestor_id(dnsdist_ffi_dnsquestion_t* dq, const char* value, size_t valueSize)
+{
+  if (!dq || !dq->dq || !value) {
+    return;
```

Comment:
I do not like this "do nothing silently" much. Wouldn't be better to indicate failure to set with a bool return value? I see that this void return value for setters is common, so something for the future to consider. Another thing to consider: the testing for `dq` and `dq->value` is not done in all setters.

Reply:
I agree this isn't great. We have added these checks after noticing that crashing dnsdist because of a typo in the Lua code was terrible (a `nil` value becomes a `nullptr`).
Yielding an exception in the FFI API doesn't work well at the moment, so we could indeed return a boolean instead, but it would make handling the potential errors in the Lua code quite cumbersome.